pay/pubspec.yaml

```diff
@@ -34,7 +34,9 @@ dependencies:
     sdk: flutter
   flutter_localizations:
     sdk: flutter
-  pay_android: ^1.0.9
+  # pay_android: ^1.1.0
```

Comment:
Testing does not require of this setup anymore, since the `example` application has dependency overrides.
Please revert to `pay_android: ^1.1.0`

Reply:
Fixed in 5bfa1a6389dd13c70f4b4982db7f9d02ffa176b3